### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by manlycode**
* Eagle PCB Schematic
* Flow Meter Data sheets

bender-worksighted

### CODE REVIEW

File: doc/pi_setup.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+$ ssh pi@<ip address>
+```
+
+The default password for this user is `raspbian`.
+
+
+## Update/Upgrade the System
+```
+$ sudo apt-get update
+$ sudo apt-get upgrade
```
Comment (albus522) [inline doc/pi_setup.md]: `sudo apt-get dist-upgrade` is generally better here

Comment (albus522) [inline doc/pi_setup.md]: Looks like you duplicate this down in system setup, so probably keep one or the other

File: doc/pi_setup.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+# change "peer" to "trust"
+
+sudo /etc/init.d/postgresql reload
+psql -U postgres
+# CREATE ROLE bender_production;
+# CREATE DATABASE bender_production;
+
+git clone https://github.com/mruby/mruby.git
+cd mruby/
+git checkout 1.2.0
```
Comment (albus522) [inline doc/pi_setup.md]: Probably include the gem setup here

File: doc/pi_setup.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+./minirake
+./bin/mruby -v
+
+# Not sure where these are from
+bunzip2 -c bender_production_2016-07-07.sql.bz2 | psql -U postgres bender_production
+
+
+# Set up Rails app
+cd /app
+bundle install
```
Comment (albus522) [inline doc/pi_setup.md]: Not sure what this is for

File: doc/pi_setup.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+sudo gem i passenger
+
+sudo passenger-install-nginx-module
+wget http://downloads.sourceforge.net/project/pcre/pcre/8.32/pcre-8.32.tar.gz
+
+# Is this needed?
+sha256sum pcre-8.32.tar.gz 
+mv pcre-8.32.tar.gz pcre.tar.gz
+cd /usr/local/lib/ruby/gems/2.2.0/gems/passenger-5.1.7
```
Comment (albus522) [inline doc/pi_setup.md]: Here up was stuff to fix a botched passenger gem. `sudo passenger-install-nginx-module` is usually all that is needed

File: doc/pi_setup.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+./minirake
+./bin/mruby -v
+
+# Not sure where these are from
+bunzip2 -c bender_production_2016-07-07.sql.bz2 | psql -U postgres bender_production
+
+
+# Nginx
+sudo passenger-install-nginx-module
```
Comment (albus522) [inline doc/pi_setup.md]: You need `sudo gem install passenger` before this

**Comment by manlycode** [comment]
Merging. I updated the .md file as per @albus522 comments.

File: doc/pi_setup.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+./bin/mruby -v
+
+# Not sure where these are from
+bunzip2 -c bender_production_2016-07-07.sql.bz2 | psql -U postgres bender_production
+
+
+# Nginx
+sudo gem install passenger
+sudo passenger-install-nginx-module
+wget http://downloads.sourceforge.net/project/pcre/pcre/8.32/pcre-8.32.tar.gz
```
Comment (albus522) [inline doc/pi_setup.md]: This `wget` usually isn't needed

**Pull request merged**